demo/tests/test_form_from_schema.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+from django.test import TestCase
+from django import forms
+
+from formidable.constants import REQUIRED
+from formidable.forms import (
+    FormidableForm, fields, get_dynamic_form_class_from_schema
+)
+from formidable.forms import widgets
+from formidable.validators import (GTEValidator, MinLengthValidator)
+from formidable.serializers.forms import ContextFormSerializer
+
+
+class TestFormFromSchema(TestCase):
+
+    def test_charfield(self):
+        class TestCharField(FormidableForm):
+            """ Test charfield """
+            charfield = fields.CharField()
+
+        formidable = TestCharField.to_formidable(label='label')
+        schema = ContextFormSerializer(instance=formidable, context={
+            'role': 'jedi'
+        }).data
+        form = get_dynamic_form_class_from_schema(schema)()
```

Comment:
From what I understood: `get_dynamic_form_class_from_schema(schema)` is a class, so `get_dynamic_form_class_from_schema(schema)()` is an instance.